pkg/agent/handler/accessrequest.go

```diff
@@ -47,18 +48,19 @@ func (h *accessRequestHandler) Handle(action proto.Event_Type, meta *proto.Event
 		return nil
 	}
 
+	log.Infof("%s event for AccessRequest", action.String())
```

Comment:
This would be better as Debug

Reply:
Yeah. They will be removed before we are done.